### PR TITLE
Update internal command list

### PR DESCRIPTION
**Pull request by karthik2804**
Updating the list of internal command names that need to be checked when installing plugins.

### CODE REVIEW

**Review by kate-goldenring — APPROVED** [review]
Thank you @karthik2804!

File: crates/plugins/src/lib.rs [hunk 1]
```diff
@@ -9,13 +9,19 @@ pub use store::PluginStore;
...
     "add",
+    "login",
     "deploy",
     "build",
     "plugin",
+    "plugins",
     "trigger",
     "external",
+    "doctor",
+    "registry",
```
Comment (itowlson) [inline crates/plugins/src/lib.rs:25]: `oci` is still accepted too. Yes, I had forgotten about it too.

**Review by itowlson — APPROVED** [review]
Good catch - thanks for this!  One tiny obscure nit it would be good to add, but not a blocker.

File: crates/plugins/src/lib.rs (hunk 1 above)
Comment (karthik2804) [reply crates/plugins/src/lib.rs:25]: added

**Review by michelleN — APPROVED** [review]

**Pull request merged**